auth/auth/auth.py

```diff
@@ -755,6 +756,21 @@ async def on_cleanup(app):
         await app['client_session'].close()
 
 
+class AuthAccessLogger(AccessLogger):
+    def __init__(self, logger: logging.Logger, log_format: str):
+        super().__init__(logger, log_format)
+        self.exclude = [
+            ('GET', re.compile('/api/v1alpha/userinfo')),
+        ]
+
+    def log(self, request, response, time):
+        for method, path_expr in self.exclude:
+            if path_expr.fullmatch(request.path) and method == request.method:
```

Comment:
I messed up this and was going to fix in another PR if you want to do the same here. `fullmatch` doesn't work for dev and pr namespaces. I was going to try with an end anchor and `match` or just set it to `match`.

Reply:
I fixed all three.